web/src/lib/components/forms/library-exclusion-pattern-form.svelte

```diff
@@ -17,7 +17,8 @@
   });
 
   $: isDuplicate = exclusionPattern !== null && exclusionPatterns.includes(exclusionPattern);
-  $: canSubmit = exclusionPattern !== '' && exclusionPattern !== null && !exclusionPatterns.includes(exclusionPattern);
+  $: canSubmit =
+    exclusionPattern !== '' && exclusionPattern !== undefined && !exclusionPatterns.includes(exclusionPattern);
```

Comment:
Since both `''` and `undefined` are falsy you could probably just write
```suggestion
  $: canSubmit = exclusionPattern && !exclusionPatterns.includes(exclusionPattern);
```

Reply:
Good call, that's much simpler!